### ISSUE TITLE
IOError: [Errno 2] No such file or directory

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#954]
Runtime error below.  This occurred when I checked out a working project into a new directory.  Log file attached.

```
#!pytb


# Sphinx version: 1.1.2
# Python version: 2.6.5
# Docutils version: 0.8.1 release
# Jinja2 version: 2.6

Traceback (most recent call last):
  File "/usr/local/lib/python2.6/dist-packages/sphinx/cmdline.py", line 189, in main
    app.build(force_all, filenames)
  File "/usr/local/lib/python2.6/dist-packages/sphinx/application.py", line 204, in build
    self.builder.build_update()
  File "/usr/local/lib/python2.6/dist-packages/sphinx/builders/__init__.py", line 196, in build_update
    'out of date' &#37; len(to_build))
  File "/usr/local/lib/python2.6/dist-packages/sphinx/builders/__init__.py", line 252, in build
    self.write(docnames, list(updated_docnames), method)
  File "/usr/local/lib/python2.6/dist-packages/sphinx/builders/__init__.py", line 291, in write
    doctree = self.env.get_and_resolve_doctree(docname, self)
  File "/usr/local/lib/python2.6/dist-packages/sphinx/environment.py", line 1225, in get_and_resolve_doctree
    doctree = self.get_doctree(docname)
  File "/usr/local/lib/python2.6/dist-packages/sphinx/environment.py", line 1208, in get_doctree
    f = open(doctree_filename, 'rb')
IOError: [Errno 2] No such file or directory: '/home/jbauer/Projects/Minerva2/Presentation20120702/_build/doctrees/PatientResponse/NashGeorge.doctree'

```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/954
- Originally reported by: Jeff Bauer
- Originally created at: 2012-06-25T15:41:23.426

**Comment by shimizukawa** [comment]
_From [nagelfar83](https://bitbucket.org/nagelfar83) on 2014-01-10 15:54:32+00:00_

Has nobody found a solution for this?

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-01-10 19:13:53+00:00_

Hmm, I'd have to know more about the original error. This shouldn't happen for a clean new build.

**Comment by shimizukawa** [comment]
_From Jeff Bauer on 2014-01-10 19:33:10+00:00_

My recollection was that I had to manually create the directory, then everything built properly.  [nagelfar83](https://bitbucket.org/nagelfar83) was that your experience?

**Comment by shimizukawa** [comment]
_From [nagelfar83](https://bitbucket.org/nagelfar83) on 2014-01-10 21:29:35+00:00_

[Jeff Bauer](https://bitbucket.org/rubic) well still not getting it to work properly i had to extract the grammar.txt and hardcode the path in the *.py file now that error is gone but sphinx is still not working as it should.

What did you mean with manually create the directory?

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-01-19 16:39:06+00:00_

Should be fixed by d9a1814088ab.
